Classes/Hooks/WizardItemsHook.php

```diff
@@ -29,10 +29,10 @@ class WizardItemsHook implements NewContentElementWizardHookInterface
      */
     public function manipulateWizardItems(&$wizardItems, &$parentObject)
     {
-        $pageId = $parentObject->id;
+        $pageId = (int) GeneralUtility::_GET('id');
```

Comment:
Please use \TYPO3\CMS\Core\Utility\GeneralUtility::_GP to retrieve the correct value, as in the NewContentElementController.
Please remove the superfluous space between (int) and code.